src/rqt_plot/plot_widget.py

```diff
@@ -155,11 +155,12 @@ def is_plottable(node, topic_name):
 class PlotWidget(QWidget):
     _redraw_interval = 40
 
-    def __init__(self, node, initial_topics=None, start_paused=False):
+    def __init__(self, node, spinner, initial_topics=None, start_paused=False):
```

Comment:
Would it be better to pass in the context so that future changes like this do not change api? I could be convinced either way.